Sources/MagicSDK/Core/Magic.swift

```diff
@@ -32,28 +32,30 @@ public class Magic: NSObject {
     /// - Parameters:
     ///   - apiKey: Your client ID. From https://dashboard.Magic.com
     ///   - ethNetwork: Etherum Network setting (ie. mainnet or goerli)
-    ///   - customNode: A custom RPC node 
-    public convenience init(apiKey: String, ethNetwork: EthNetwork, locale: String = Locale.current.identifier) {
-        self.init(urlBuilder: URLBuilder(apiKey: apiKey, network: ethNetwork, locale: locale))
+    ///   - customNode: A custom RPC node
+    ///   - viewHostProvider: An optional `UIViewController` provider for login views to embed within
```

Comment:
Given what I described [here](https://github.com/magiclabs/magic-ios/pull/39#issuecomment-1984363843) could we potentially isolate this parameter to a single convenience initializer? This is, of course, assuming you will not need to pass an `EthNetwork` configuration with your setup.

Reply:
Of course, in our use case we only provide the apiKey so whatever pattern you think makes sense here.